am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -351,4 +371,24 @@ private DefaultPermissionGrant createDefaultPermissionGrant(String roleName,
             .attributePermissions(createPermissionsForAttribute(JsonPointer.valueOf(attribute), permissions, PUBLIC))
             .build();
     }
+
+
+
+    private Map<JsonPointer, SecurityClassification> getJsonPointerStringMap() {
+        Map<JsonPointer, SecurityClassification> securityClassificationMap = new ConcurrentHashMap<>();
+        securityClassificationMap.put(JsonPointer.valueOf(""), PUBLIC);
+        securityClassificationMap.put(JsonPointer.valueOf(NAME_ATTRIBUTE), PUBLIC);
+        securityClassificationMap.put(JsonPointer.valueOf(CITY_ATTRIBUTE), PUBLIC);
+        return securityClassificationMap;
+    }
+
+    private Map<JsonPointer, SecurityClassification> createJsonPointerStringMap(String... args) {
```

Comment:
same as above

Reply:
done...Avoided overloaded method here as it causes problem with lamda